### ISSUE TITLE
Create aa way to make submission of HTP flat files easier 

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1859]
https://www.pombase.org/submit-data/bulk-annotation

**Comment by ValWood** [comment]
https://www.youtube.com/watch?v=nyNp09WYLzw&ab_channel=ISCB ?

**Comment by kimrutherford** [comment]
> would this be a possible LinkML project

I don't think so.  As I understand it, LinkML is a way of describing data.  We need a way to upload data files and to then store/manage the files.

**Comment by ValWood** [comment]
I think you can use a LinkML data structure/ associated Tools to create input forms to match the data model. This might not be helpful though.
